exercises/robot-simulator/lib/orientation.dart

```diff
@@ -0,0 +1 @@
+enum Orientation { NORTH, EAST, SOUTH, WEST }
```

Comment:
Recommend: Change to lowerCamelCase
```suggestion
enum Orientation { north, east, south, west }
```

In dart 1: UPPERCASE was preferred style for constants. In dart 2 its lowerCamelCase

https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names